docker/oso-image-inspector/centos7/root/config.yml

```diff
@@ -53,14 +53,11 @@
     set_fact:
       temp_name: "{{ kubeletArguments['node-labels'] }}"
 
-  - name: print nodename
-    debug:
-      msg: "{{ temp_name[4].split('=')[1] }}"
-
   - name: populate templated config file
     copy:
       content: |
         node_hostname: {{ temp_name[4].split('=')[1] }}
+        cluster_name: {{ temp_name[4].split('=')[1].split('-node')[0] }}
```

Comment:
This is fragile and liable to break in dedicated with random cluser names.  Can you use {{ oo_clusterid }} instead?